docs/nidcpower/enums.rst

```diff
@@ -91,6 +91,56 @@ ComplianceLimitSymmetry
 
 
 
+CurrentLevelAutorange
+---------------------
+
+.. py:class:: CurrentLevelAutorange
+
+    .. py:attribute:: CurrentLevelAutorange.OFF
```

Comment:
I think for these sort of attribute we decided to expose as Boolean in nimi-python.
I'm surprised they didn't call this NIDCPOWER_ATTR_CURRENT_LEVEL_AUTORANGE_ENABLED. I'll circle back to the team.

Reply:
Applies to all 4.

Reply:
We shipped that way? From our guidelines:

> Don't use enums when a bool will do
> C uses ViBoolean but provides named constants. Don't do that for Python
> LabVIEW APIs provide ring controls or enums. Don't do that for Python
> We think this is more obvious for Python clients

https://github.com/ni/nimi-python/wiki/API-design-guidelines#enums